.github/workflows/node-ci-pr.yml

```diff
@@ -44,22 +33,18 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-latest]
```

Comment:
This did not have the `node-version` and so was creating a cache key starting with just `Linux-`